virttest/qemu_devices/qcontainer.py

```diff
@@ -599,15 +609,17 @@ def simple_hotplug(self, device, monitor, bus=None):
 
         if isinstance(device, qdevices.QDevice):
             if bus is None:
-                bus = self.get_buses({'aobject': 'pci.0'})[0]
+                if self.is_pci_device(device._get_alternative_name()):
```

Comment:
You could get device name by device["driver"].

Reply:
@huangyum @luckyh Updated to `device['driver']` to avoid private function calling, thanks.